api/src/main/resources/emails/welcomeemail/content.html

```diff
@@ -95,23 +92,79 @@
       text-decoration: none;
       cursor: pointer;
     }
+    th.short {
+      width: 20%;
+    }
+    th.long {
+      width: 80%
+    }
   </style>
 </head>
 <body>
-  <img src="${HEADER_IMG}">
-  <div class="welcome-message">Welcome to the All of Us Researcher Workbench!</div>
-  <div><div class="text user-field">Your Username: </div><div class="user-info">${USERNAME}</div></div>
-  <div><div class="text user-field">Your Temporary Password: </div><div class="user-info">${PASSWORD}</div></div>
-  <br>
-  <div class="text">Quick guide on how to get started:</div>
-  <a class="card" href="https://myaccount.google.com/signinoptions/two-step-verification">
-    <img class="bullet" src="${BULLET_1}">
-    <div class="card-text"><div class="link">Sign into Google</div> with your new account, change your password, and set up 2-Step Verification</div>
-  </a>
-  <a class="card" href="https://myaccount.google.com/intro/recovery/email?hl=en-US">
-    <img class="bullet" src="${BULLET_2}">
-    <div class="card-text"><div class="link">Add</div> your recovery email address</div>
-  </a>
-  <div class="after text">After these steps are completed, you are ready to use the Researcher Workbench</div>
-  <a class="button" href="${URL}">Get Started</a>
+  <table>
+    <thead>
+      <tr>
+        <th class="short">&nbsp;</th>
+        <th class="short">&nbsp;</th>
+        <th class="long">&nbsp;</th>
+      </tr>
+    </thead>
+    <tr>
+      <td colspan="3">
+        <img src="${HEADER_IMG}">
+      </td>
+    </tr>
+    <tr>
+      <td colspan="3">
+        <div class="welcome-message">Welcome to the All of Us Researcher Workbench!</div>
+      </td>
+    </tr>
+    <tr>
+      <td colspan="2">
+        <div class="text user-field">Your Username: </div>
+      </td>
+      <td>
+        <div class="user-info">${USERNAME}</div>
+      </td>
+    </tr>
+    <tr>
+      <td colspan="2">
+        <div class="text user-field">Your Temporary Password: </div>
+      </td>
+      <td>
+        <div class="user-info">${PASSWORD}</div>
+      </td>
+    </tr>
+    <tr>
+      <td colspan="3">
+        <div class="text">Quick guide on how to get started:</div>
+      </td>
+    </tr>
+    <tr class="card" href="https://myaccount.google.com/signinoptions/two-step-verification">
```

Comment:
I was imagining that just this section (bullets + steps) would be in the table. If you had things this way, then perhaps the table autosizing would work properly and avoid the padding?

Reply:
You think just that section? I can try two tables as well, one for the bullets, and one for username/password.